pyproject.toml

```diff
@@ -76,18 +74,16 @@ max-complexity = 18
 known-first-party = ["genai"]
 
 [tool.poetry.group.dev.dependencies]
+python-dotenv = "^1.0.0"
```

Comment:
Probably not worth the effort now, just a note:

Can we somehow make sure that production code does not depend on dev packages? 🤔
I think we'd have to extract "test" dependencies into a separate group and be more careful when writing tests -> e.g. use extra marker for "dev" tests (e.g. testing scripts)

Reply:
I will do that in the separate PR.